packages/salesforcedx-vscode-apex/package.json

```diff
@@ -25,7 +25,7 @@
     "Programming Languages"
   ],
   "dependencies": {
-    "@salesforce/apex-node": "0.1.15",
+    "@salesforce/apex-node": "0.1.16",
```

Comment:
This will need to be updated again once the next version of the library is released